src/DataFrames.jl

```diff
@@ -84,6 +85,7 @@ export @~,
        unique!,
        unstack,
        writetable,
+       encodings,
```

Comment:
I don't think it's worth exporting this function. Just use `StringEncodings.encodings` in the error message. Also better not add `using StringEncodings` just for two calls: use the fully qualified names in these places.